api/api.go

```diff
@@ -665,7 +673,71 @@ func (s *BlockChainAPI) Call(
 	overrides *StateOverride,
 	blockOverrides *BlockOverrides,
 ) (hexutil.Bytes, error) {
-	return hexutil.Bytes{0, 1, 2, 3, 4, 5, 6, 7, 8, 9}, nil
+	script := `
+        import EVM from 0xf8d6e0586b0a20c7
+
+        access(all)
+        fun main(data: [UInt8], contractAddress: [UInt8; 20]): [UInt8] {
+            let flowAccount = getAuthAccount<auth(Storage) &Account>(Address(0xf8d6e0586b0a20c7))
+            let bridgedAccount = flowAccount.storage.borrow<&EVM.BridgedAccount>(
+                from: /storage/evm
+            ) ?? panic("Could not borrow reference to the bridged account!")
+
+            let evmResult = bridgedAccount.call(
+                to: EVM.EVMAddress(bytes: contractAddress),
+                data: data,
+                gasLimit: 300000,
+                value: EVM.Balance(flow: 0.0)
+            )
+
+            return evmResult
+        }
```

Comment:
I would extract this into a `call.cdc` file and then embed it.

Reply:
Ah, good point :+1: 
I have updated that in https://github.com/onflow/flow-evm-gateway/pull/24/commits/86a9487880b2ed94b91cb5a0e7d02ff48cc7760d and added some TODOs for interpolating some parts of the script.